script.js

```diff
@@ -0,0 +1,93 @@
+var options = document.querySelectorAll(".switcher__option");
+var squares = document.querySelectorAll(".switcher__option-square");
+
+options[0].addEventListener("click", function() {
+  options[0].classList.add("active-div");
+  options[1].classList.remove("active-div");
+  options[2].classList.remove("active-div");
+  squares[0].classList.add("active-square");
+  squares[1].classList.remove("active-square");
+  squares[2].classList.remove("active-square");
+  drawCanvas(4);
+});
+
+options[1].addEventListener("click", function() {
+  options[1].classList.add("active-div");
+  options[0].classList.remove("active-div");
+  options[2].classList.remove("active-div");
+  squares[1].classList.add("active-square");
+  squares[0].classList.remove("active-square");
+  squares[2].classList.remove("active-square");
+  drawCanvas(32);
+});
+
+options[2].addEventListener("click", function() {
+  options[2].classList.add("active-div");
+  options[0].classList.remove("active-div");
+  options[1].classList.remove("active-div");
+  squares[2].classList.add("active-square");
+  squares[0].classList.remove("active-square");
+  squares[1].classList.remove("active-square");
+  drawCanvas(256);
+});
+
+const firstPic = [
+  ["00BCD4", "FFEB3B", "FFEB3B", "00BCD4"],
+  ["FFEB3B", "FFC107", "FFC107", "FFEB3B"],
+  ["FFEB3B", "FFC107", "FFC107", "FFEB3B"],
+  ["00BCD4", "FFEB3B", "FFEB3B", "00BCD4"]
+];
+
+secondPic.forEach(row => {
+  row.forEach(cell => {
+    cell[3] /= 255;
+  });
+});
+
+function drawCanvas(size) {
```

Comment:
The function is too big and difficult to work with. How about moving some logic to new functions? 
```javascript
function drawCanvas(size) {
  // prepare some variables
  // ...
  switch (size) {
      case 4:
        activateFirstImage(ctx);
        break;
      case 32:
        activateSecondImage(ctx);
        break;
      case 256:
        activateThirdImage();
        break;
  }
}
```